easybuild/easyconfigs/p/phASER/phASER-30666e7.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PackedBinary'
+
+name = 'phASER'
+version = '30666e7'
+
+homepage = "https://github.com/secastel/phaser/"
+description = """Detection of Regulatory DNA Sequences using GRO-seq Data.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/secastel/phaser/archive']
+sources = ['%s.zip' %version]
+
+
+
+builddependencies = (
+    ('Cython','0.22-foss-2015b-Python-2.7.9'),
+)
+
+
+
+dependencies = [('SAMtools','1.2-foss-2015b'),
```

Comment:
Do not put toolchain in version suffix. E.g.:

('SAMtools','1.2-foss-2015b'), -> ('SAMtools', '1.2' , '', ('foss', '2015b')),

('PyVCF', '0.6.7-foss-2015b-Python-2.7.9'), -> ('PyVCF', '0.6.7', '-Python-2.7.9', ('foss', '2015b')),

etc.